src/actions/commands/actions.ts

```diff
@@ -1010,8 +1010,10 @@ class CommandOverrideCopy extends BaseCommand {
         .map(range => {
           const start = Position.EarlierOf(range.start, range.stop);
           const stop = Position.LaterOf(range.start, range.stop);
-          return vimState.editor.document.getText(new vscode.Range(start, stop.getRight()));
-        })
+          return start < stop
```

Comment:
The docs for `EarlierOf` state:

```
 /**
   * Returns which of the 2 provided Positions comes earlier in the document.
   */
```

so start *should* always be >= stop. 

What's the case when `start < stop`?